spring-petclinic-customers-service/src/main/java/org/springframework/samples/petclinic/customers/CustomersServiceApplication.java

```diff
@@ -26,7 +29,13 @@
 @SpringBootApplication
 public class CustomersServiceApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(CustomersServiceApplication.class, args);
-	}
+    @Autowired
+    private OwnerRepository ownerRepository;
+
+    @Autowired
+    private PetRepository petRepository;
```

Comment:
Do these need to be here? Isn't the OwnerRepository / PetRepository not already annotated with @Repository?

Reply:
Had followed the Cosmosdb sample that had them. I have now removed them and it works fine.